win/PyTango_VS9/PyTango.vsprops

```diff
@@ -90,30 +90,30 @@
         Description="[PyTango]: Starting python $(WinSetupArgs)"
     />
 
-    <UserMacro Name="PyTangoBaseFile" Value="_PyTango" />
+    <UserMacro Name="PyTangoBaseFile" Value="_tango" />
     
-    <UserMacro Name="TangoMajorVer" Value="8" />
-    <UserMacro Name="TangoMinorVer" Value="1" />
+    <UserMacro Name="TangoMajorVer" Value="9" />
+    <UserMacro Name="TangoMinorVer" Value="2" />
     <UserMacro Name="TangoPatchVer" Value="2" />
     <UserMacro Name="TangoVer" Value="$(TangoMajorVer).$(TangoMinorVer).$(TangoPatchVer)" />

```

Comment:
@NexeyaSGara See what I wrote in the [readme](https://github.com/tango-cs/PyTango/tree/tango9#requirements):

> As a general rule, libtango and pytango should share the same major and minor version (for a version X.Y.Z, X and Y should match)